packages/platform-core/API.md

```diff
@@ -29,13 +43,52 @@ export enum CDKContextKey {
     DEPLOYMENT_TYPE = "amplify-backend-type"
 }
 
+// @public
+export class CwdPackageJsonLoader {
+    constructor(fs?: typeof _fs, fsp?: typeof _fsp);
+    loadCwdPackageJson: () => Promise<PackageJson>;
+}
+
+// @public (undocumented)
+export type DeploymentTimes = {
+    synthesisTime?: number;
+    totalTime?: number;
+};
+
 // @public
 export class FilePathExtractor {
     constructor(stackTraceLine: string);
     // (undocumented)
     extract: () => string | undefined;
 }
 
+// @public
+export class LibraryVersionFetcher {
+    // (undocumented)
+    fetch: (absolutePackageJsonPath: string) => string;
+}

```

Comment:
I'd create `PackageJsonReader` instead. If we have to read whole thing to get version we can as well serve all properties we care about and caller will select what they want.

`CwdPackageJsonLoader` could then use `PackageJsonReader` internally (assuming it does some trick with process.cwd`.

Reply:
Refactored `PackageJsonReader`